### ISSUE TITLE
Date and time picker/s for the assignment metadata

### ISSUE THREAD

**Issue by lgpage** [jupyter/nbgrader#809]
It may be more user friendly to have some sort of date and/or time picker for when setting an assignment's due-date.

**Comment by jhamrick** [comment]
Yeah, I was thinking something similar, thanks for opening this issue!

This will probably require adding some javascript library but there's bound to be a plugin for jquery or something that does this.

**Comment by jhamrick** [comment]
See also #809 , which requests a way of choosing the timezone too.

**Comment by perllaghu** [comment]
Time-picker - may be simply just a case of marking the input as `type="Date"` - see https://html.spec.whatwg.org/#the-input-element
